doc/ubuntu_release_process.md

```diff
@@ -7,6 +7,8 @@ covers releases xenial+.
 Ubuntu packaging is stored as branches in the upstream cloud-init
 repo.  For example, see the ``ubuntu/devel``, ``ubuntu/xenial`` ... branches in the [upstream git repo](https://git.launchpad.net/cloud-init/).  Note that changes to the development release are always done in ubuntu/devel, not ubuntu/<release-name>.
 
+To suppport daily recipe builds, there are related ``ubuntu/daily/devel``, ``ubuntu/daily/xenial``, ... branches which are used to support our daily recipe package builds. The ubuntu/daily/$release branches are snapshots of ubuntu/$release branches with any cherry picks reverted so that our build recipe can merge tip of master into ubuntu/$release without merge conflicts.
```

Comment:
We build cloud-init master with each release branch to provide a daily cloud-init for each of the releases we support.  Certain features and behavior changes are disabled in release branches.  These release branch changes prevent master from merging with the release branch and break the daily recipe builds.  To resolve this conflict we fixes (reverting cherry picks or other feature redactions) to the release branches in a  separate branch named ``ubuntu/daily/<release>`` branch.  For example ``ubuntu/daily/devel``, ``ubuntu/daily/xenial``.

Reply:
thanks. pulled this in.

Reply:
Due to later comments, I consolidated this suggestion a bit and placed details about ubuntu/daily/devel below in the Daily packaging recipes section.